core/src/main/java/it/pagopa/selfcare/mscore/core/ContractService.java

```diff
@@ -296,6 +293,10 @@ private InstitutionToNotify toInstitutionToNotify(Institution institution) {
         try {
             InstitutionProxyInfo institutionProxyInfo = partyRegistryProxyConnector.getInstitutionById(institution.getExternalId());
             toNotify.setIstatCode(institutionProxyInfo.getIstatCode());
+            GeographicTaxonomies geographicTaxonomies = partyRegistryProxyConnector.getExtByCode(toNotify.getIstatCode());
+            toNotify.setCounty(geographicTaxonomies.getProvinceAbbreviation());
+            toNotify.setCountry(geographicTaxonomies.getCountryAbbreviation());
+            toNotify.setCity(geographicTaxonomies.getDescription().replace(" - COMUNE", ""));
```

Comment:
you could replace hardcoded string with a constant, es. REGEX_DESCRIPTION_TO_REPLACED = " - COMUNE"